assets/sass/protocol/base/elements/_titles.scss

```diff
@@ -23,6 +24,9 @@ h6 {
 
     @supports (--css: variables) {
         font-family: var(--title-font-family);
+        font-variant-ligatures: var(--title-font-ligatures);
```

Comment:
https://github.com/mozilla/bedrock/pull/15790#discussion_r1910681933

just noticed this comment on another PR, maybe best to use `none` as default here as well